exonum-java-binding/core/rust/exonum-java/README.md

```diff
@@ -0,0 +1,16 @@
+# Exonum Java Binding Application
```

Comment:
Why don't we use the one from the `exonum-java-binding/`?

Reply:
The only reason is that it contains a link to cryptocurrency demo which will be obviously invalid in case of packaged App